### ISSUE TITLE
mis-housed card

### ISSUE THREAD

**Issue by Gryphavik** [jeremylarner/keyteki#299]
I had a Mars card mis identified as a Dis card. The whole game ended up going buggy and we quit it.

**Comment by samlehman** [comment]
Do you remember the deck name?

**Comment by Gryphavik** [comment]
no, sorry. I should have captured it. The card was Tunk. It seemed to switch houses while it was in my hand. It showed the Dis symbol superimposed over the Mars symbol.

**Comment by samlehman** [comment]
It's possible it was a maverick card, which is a card from one house that appears in another.

Do you remember the other two houses?

[There exists 29 decks with a Maverick Dis Tunk](https://keyforge-compendium.com/decks?utf8=%E2%9C%93&filterrific%5Bwith_deck_name%5D=&filterrific%5Bwith_card_ids%5D%5B%5Dcard_id=499&filterrific%5Bwith_card_ids%5D%5B%5Dcard_type=%3E%3D&filterrific%5Bwith_card_ids%5D%5B%5Dcard_count=1&filterrific%5Bwith_adhd%5Da_rating=&filterrific%5Bwith_adhd%5Da_type=%3E&filterrific%5Bwith_adhd%5Db_rating=&filterrific%5Bwith_adhd%5Db_type=%3E&filterrific%5Bwith_adhd%5Dc_rating=&filterrific%5Bwith_adhd%5Dc_type=%3E&filterrific%5Bwith_adhd%5De_rating=&filterrific%5Bwith_adhd%5De_type=%3E&filterrific%5Bwith_adhd%5Dconsistency_rating=&filterrific%5Bwith_adhd%5Dconsistency_type=%3E&filterrific%5Bwith_sas%5Dsas_rating=&filterrific%5Bwith_sas%5Dsas_type=%3E&filterrific%5Bwith_sas%5Dcards_rating=&filterrific%5Bwith_sas%5Dcards_type=%3E&filterrific%5Bwith_sas%5Dsynergy_rating=&filterrific%5Bwith_sas%5Dsynergy_type=%3E&filterrific%5Bwith_sas%5Danti_synergy_rating=&filterrific%5Bwith_sas%5Danti_synergy_type=%3E&filterrific%5Bwith_power_level%5Dcount=&filterrific%5Bwith_power_level%5Dtype=%3E%3D&filterrific%5Bwith_chains%5Dcount=&filterrific%5Bwith_chains%5Dtype=%3E%3D&filterrific%5Bwith_wins%5Dcount=&filterrific%5Bwith_wins%5Dtype=%3E%3D&filterrific%5Bwith_losses%5Dcount=&filterrific%5Bwith_losses%5Dtype=%3E%3D&filterrific%5Bwith_card_count%5Dcreature_count=&filterrific%5Bwith_card_count%5Dcreature_type=%3E&filterrific%5Bwith_card_count%5Daction_count=&filterrific%5Bwith_card_count%5Daction_type=%3E&filterrific%5Bwith_card_count%5Dartifact_count=&filterrific%5Bwith_card_count%5Dartifact_type=%3E&filterrific%5Bwith_card_count%5Dupgrade_count=&filterrific%5Bwith_card_count%5Dupgrade_type=%3E&filterrific%5Bwith_rare_count%5Dcount=&filterrific%5Bwith_rare_count%5Dtype=%3E%3D&filterrific%5Bwith_common_count%5Dcount=&filterrific%5Bwith_common_count%5Dtype=%3E%3D&filterrific%5Bwith_uncommon_count%5Dcount=&filterrific%5Bwith_uncommon_count%5Dtype=%3E%3D&filterrific%5Bwith_fixed_count%5Dcount=&filterrific%5Bwith_fixed_count%5Dtype=%3E%3D&filterrific%5Bwith_variant_count%5Dcount=&filterrific%5Bwith_variant_count%5Dtype=%3E%3D&filterrific%5Bwith_maverick_count%5Dcount=&filterrific%5Bwith_maverick_count%5Dtype=%3E%3D&filterrific%5Bby_house%5D%5Bhouse_ids%5D%5B%5D=6&filterrific%5Bby_house%5D%5Border_type%5D=Includes&filterrific%5Bsorted_by%5D=consistency_desc&commit=Filter+decks&filterrific%5Bsorted_by%5D=consistency_desc)

**Comment by Gryphavik** [comment]
I know it was Dis and Mars. I;m not certain of the other house; it was a sealed game, and that is the only way I play, so I see a lot of decks. It was probably Brobnar or Untamed.

**Comment by Gryphavik** [comment]
Ok, so it is a known issue. I found a maverick Titan Mechanic as well. Deck: Mrs. "Lynx" Qeev. The image won't display...
![image](https://user-images.githubusercontent.com/48365135/54057445-79c3ee80-419f-11e9-8d98-657b782f63bb.png)

**Comment by Gryphavik** [comment]
Mechanic is acting as a Dis card as well

**Comment by samlehman** [comment]
In that case, the Maverick Titan is a Dis card: https://keyforge-compendium.com/decks/058f69d7-8019-41ed-920d-28b655bd1ac3

I've had similar issues with the image not working correctly for Maverick cards before too.

**Comment by Gryphavik** [comment]
ok. just trying to be a helpful member of the community.
